### DISCUSSION TITLE
Special large grain size

### DISCUSSION THREAD

**Discussion by Abaqus-Voronoi-toolbox** [neperfepx/neper#786] [Q&A] [answered]
Hi prof. Quey,
I want to generate a cubic model (1*1*1) with a series of special large grain (see the below figure in your paper), could you help me? Thank you in advance! 
![image](https://github.com/neperfepx/neper/assets/150608850/484dee99-ef32-422a-ae5d-b805172ec13e)
Regards.

**Comment by rquey** [comment, marked as answer]
This discussion has been closed as obsolete. Please repost if necessary.
